README.md

```diff
@@ -24,24 +24,17 @@ The fields in the table below can be used in these parts of fiboa documents:
-| ml:verified_from | datetime | **REQUIRED**. the first date for which the boundary is verified to be present on ground (e.g. through ground-truth or remote sensing image verification) |
-| ml:verified_until | datetime | **REQUIRED**. the last date for which the boundary is verified to be present on ground (e.g., through ground-truth or remote sensing image verification) |
-| ml:author | string | **REQUIRED**. Name of individual or organization who created this |
-| ml:quality | string | **REQUIRED**. the level of quality control applied to this polygon |
-| ml:confidence | float | a value between zero and one that represents the confidence in a generated polygon (higher value = higher confidence) |
-| ml:machine_generated | boolean | **REQUIRED**. Indication of machine-generated boundaries or not |
-| ml:country_code | string | **REQUIRED**. [Auto-added] three-letter country code for the country that contains the field, e.g. SDN; Can be found <https://www.iso.org/obp/ui/#search> under the Alpha-3 code column |
-| ml:admin0 | string | **REQUIRED**. [Auto-added] a unique admin0 (country) name for the administrative region that contains the field - can auto-generate from [GADM boundaries](https://geodata.ucdavis.edu/gadm/) |
-| ml:admin1 | string | **REQUIRED**. [Auto-added] a unique admin1 (state) name for the administrative region that contains the field - can auto-generate from [GADM boundaries](https://geodata.ucdavis.edu/gadm/) |
-| ml:admin2 | string | **REQUIRED**. a unique admin2 (county) name for the administrative region that contains the field - can auto-generate from [GADM boundaries](https://geodata.ucdavis.edu/gadm/) |
-| ml:crop_type | string | the type of crop e.g. maize |
-| ml:crop_category | string | **REQUIRED**. high-level crop category e.g. cereal (e.g. [FAO](https://unstats.un.org/unsd/classifications/Family/Detail/1002)) |
+| ml:creation_date | date   | **REQUIRED**. Date collection was created or published |
+| ml:verified_from | date   | **REQUIRED**. the first date for which the boundary is verified to be present on ground (e.g. through ground-truth or remote sensing image verification) |
+| ml:author        | string | **REQUIRED**. Name of individual or organization who created this. |
+| ml:country_code  | string | **REQUIRED**. ISO 3166-1 alpha-3 country code. Three-letter country code for the country that contains the field, e.g. `SDN` for Sudan. Can be found at <https://www.iso.org/obp/ui/#search> under the Alpha-3 code column. |
+| ml:admin1        | string | **REQUIRED**. Modified ISO 3166-2 codes for identifying the principal subdivisions (e.g., provinces or states) of a country (i.e. admin1) that contains the field. The feature only contains the second part of the ISO 3166-2 code, to reduce redundancy. |
+| ml:admin2        | string | A unique admin2 (county) name for the administrative region that contains the field. The feature contains the last part of the HASC 2 codes. |
```

Comment:
A link to the documentation around HASC2 codes would be good.

Reply:
On it!